indico/modules/events/timetable/client/js/preprocess.ts

```diff
@@ -73,7 +76,7 @@ export function preprocessTimetableData(
             type: entryTypeMapping[_id[0]],
             id: childId,
             title: _childEntry.title,
-            startDt: moment(`${_childEntry.startDate.date} ${_childEntry.startDate.time}`),
+            startDt: dateToMoment(_childEntry.startDate),
```

Comment:
This breaks the layout for child entries, because the parent `startDate` is still calculated using ```moment(`${entry.startDate.date} ${entry.startDate.time}`)``` ([here](https://github.com/indico/indico/pull/6635/files#diff-df4a20f874b96cef4412fc10fc683b5378eccbb891e86e3e089a094c51052812R53)) so it does not have the correct timezone.
We should make all time handling logic timezone-aware but it's probably better to do it at the same time to avoid breaking things in the meantime (can be done in this PR or a separate one, up to you :))